### PR TITLE
Add CalibrePHP

### PR DESCRIPTION
**Pull request by demorfi**
Add path calibre in wizard

### CODE REVIEW

File: spk/calibrephp/src/installer.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    # Remove the web interface
+    rm -fr ${WEB_DIR}/${PACKAGE}
+
+    exit 0
+}
+
+preupgrade ()
+{
+	# Save some stuff
```
Comment (Diaoul) [inline spk/calibrephp/src/installer.sh]: Tab here, indent with spaces please

File: spk/calibrephp/src/installer.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    else
+        mv /etc/php/conf.d/${PACKAGE_NAME}.ini ${TMP_DIR}/${PACKAGE}/
+    fi
+
+    exit 0
+}
+
+postupgrade ()
+{
+	# Restore some stuff
```
Comment (Diaoul) [inline spk/calibrephp/src/installer.sh]: Same here

File: spk/calibrephp/src/installer.sh [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+		# Configure open_basedir
+		if [ "${USER}" == "nobody" ]; then
+			echo -e "<Directory \"${WEB_DIR}/${PACKAGE}\">\nphp_admin_value open_basedir ${WEB_DIR}/${PACKAGE}:${wizard_calibre_dir:=/volume1/calibre/} \n</Directory>" > /usr/syno/etc/sites-enabled-user/${PACKAGE}.conf
+		else
+			echo -e "[PATH=${WEB_DIR}/${PACKAGE}]\nopen_basedir = ${WEB_DIR}/${PACKAGE}:${wizard_calibre_dir:=/volume1/calibre/}" > /etc/php/conf.d/${PACKAGE_NAME}.ini
+		fi
+
+		# Set metadata path
+		if [ -f "${wizard_calibre_dir:=../}metadata.db" ]; then
```
Comment (Dr-Bean) [inline spk/calibrephp/src/installer.sh]: This should probably be the same defaultValue as you specify in the wizard? Same on L40.

**Comment by demorfi** [comment]
Yes, this is error.

**Comment by Dr-Bean** [comment]
I'd suggest to revert this commit: https://github.com/demorfi/spksrc/commit/dfa18b4e9e61778dc18576dd9d10220e971158a0, because I don't think that's going to work. Using tabs or spaces depends on the type of file, but mostly, spaces are used. Check other packages if you're not sure which to use.

Besides that, one question: you've forked the original repository: https://github.com/openam/calibrephp and you have an open PR to it. The last reponse of the owner (on this repository at least) seems to have been in January. Is it officially no longer maintained?

**Comment by demorfi** [comment]
Yes, most likely original repository is no longer supported.

**Comment by demorfi** [comment]
Original repository https://github.com/openam/calibrephp was updated. You propose roll back changes to use of spaces?

**Comment by Dr-Bean** [comment]
Most files use spaces (e.g. shell scripts), and Makefiles _mostly_ use tabs. Best approach for you: revert that commit. Instead, compare your package with other packages: Use spaces/tabs when appropriate, and check that the indentation is the same.

**Comment by demorfi** [comment]
Spaces and tabs are given in compare with package cops.

**Comment by Dr-Bean** [comment]
The used fork is still yours, care to change that? When that's done, I'll take another look.

**Comment by demorfi** [comment]
Yes.

**Comment by Dr-Bean** [comment]
Any updates?

**Comment by demorfi** [comment]
Yes. When will the next version of the package, I make a separate request.

**Comment by Dr-Bean** [comment]
Not sure if I'm understanding you correctly. Do you want to close this PR, and open a new PR when CalibrePHP releases a new version? Or did you mean something else?

**Comment by demorfi** [comment]
No, apparently I not understand you correctly :)
I was referring to in future updates for CalibrePHP.

**Comment by KaraokeStu** [comment]
Hi, please could you rebase and confirm this all still works ok - I'm trying to clean up old pull requests.

If I don't hear anything by 31st July 2017, I will close the PR.

Thanks,

Stuart

**Pull request closed without merging**